sdk/src/process_instruction.rs

```diff
@@ -321,6 +322,24 @@ pub mod stable_log {
         ic_logger_msg!(logger, "Program log: {}", message);
     }
 
+    /// Emit a program log data.
+    ///
+    /// The general form is:
+    ///
+    /// ```notrust
+    /// "Program log data: <program-id> <binary-data-in-base64>+"
```

Comment:
🤔 I have a couple nits about the log form:

1. The "Program log data:" feels a little wordy.  How about "Program data:" or "Program event:"?  
2. Turns out we don't need the "<program-id>" here at all because a [program invoke](https://github.com/solana-labs/solana/blob/b507715d449acd5f04b1563957e4d52392abe700/sdk/src/process_instruction.rs#L297) message is guaranteed to precede this message.  We make this same assumption to avoid repeating the program id in program log messages already.

Reply:
Good points! 

1. "Program data: " is a bit better, I agree. I'm having trouble coming up with a great name here.
2. Aha! That's a great point. I've also update the program return data entries.

Reply:
I've also renamed "Program return data: " to "Program return: "